client/src/plus/index.scss

```
@@ -79,6 +84,11 @@
         .modal-content {
           position: relative;
           z-index: 10001;
+          padding: 64px 12px;
```

We could do:

```suggestion
          padding: ($base-spacing * 2.5) math.div($base-spacing, 2);
```

> $base-spacing * 2.5 == 60px

For the division you would then need to also include the following at the top of the file:

```sass
@use "sass:math";
```